java/api/src/main/java/org/ray/api/id/JobId.java

```diff
@@ -38,11 +35,11 @@ public static JobId fromByteBuffer(ByteBuffer bb) {
     return new JobId(byteBuffer2Bytes(bb));
   }
 
-  public static JobId fromInt(int value) {
+  public static JobId fromShort(short value) {
```

Comment:
As discussed offline, 2 bytes is too short for job id. @edoakes will increase this length.
So we can still keep the value as an integer.

Reply:
Sure, we can hold this PR.